app/features/worksheet_generator/document_loaders.py

```diff
@@ -0,0 +1,372 @@
+from langchain_community.document_loaders import YoutubeLoader
+from langchain_community.document_loaders import PyPDFLoader
+from langchain_community.document_loaders.csv_loader import CSVLoader
+from langchain_community.document_loaders import TextLoader
```

Comment:
You can import the document loaders in a single line.

Reply:
Done!